<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="logButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Log</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMyViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionTime_View"/>
    <addaction name="actionxy_View"/>
    <addaction name="actionTrafficLight_View"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionen"/>
     <addaction name="actionde"/>
    </widget>
    <addaction name="actionLimits"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionChannels"/>
   </widget>
   <addaction name="menuMyViews"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTime_View">
   <property name="text">
    <string>Time View</string>
   </property>
  </action>
  <action name="actionxy_View">
   <property name="text">
    <string>xy View</string>
   </property>
  </action>
  <action name="actionTrafficLight_View">
   <property name="text">
    <string>TrafficLight View</string>
   </property>
  </action>
  <action name="actionLimits">
   <property name="text">
    <string>Limits</string>
   </property>
  </action>
  <action name="actionen">
   <property name="text">
    <string>en</string>
   </property>
  </action>
  <action name="actionde">
   <property name="text">
    <string>de</string>
   </property>
  </action>
  <action name="actionChannels">
   <property name="text">
    <string>Channels</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
